playkit/src/main/java/com/kaltura/playkit/player/AudioCodecSettings.java

```diff
@@ -0,0 +1,59 @@
+package com.kaltura.playkit.player;
+
+import com.kaltura.playkit.PKAudioCodec;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AudioCodecSettings {
+    private List<PKAudioCodec> audioCodecPriorityList = new ArrayList<>();
+    private boolean allowAudioMixedMimeTypes = false;
```

Comment:
Remove "audio" from field names
Change "MimeTypes" to "Codecs"

Reply:
Refactored the name